nextcord/ext/menus/menu_pages.py

```diff
@@ -167,14 +167,17 @@ class MenuPaginationButton(nextcord.ui.Button['MenuPaginationButton']):
     A custom button for pagination that will be disabled when unavailable.
     """
 
-    def __init__(self, style: nextcord.ButtonStyle, emoji: EmojiType):
-        super().__init__(style=style, emoji=emoji)
-        self._emoji = _cast_emoji(emoji)
+    def __init__(self, emoji: Optional[EmojiType] = None, **kwargs):
+        super().__init__(emoji=emoji, **kwargs)
+        self._emoji = _cast_emoji(emoji) if emoji else None
```

Comment:
@veni-vidi-code Functionality-wise not much of a difference, but what do you think of this instead?

```suggestion
    def __init__(self, **kwargs):
        super().__init__(**kwargs)
        emoji = kwargs.get(emoji, None)
        self._emoji = _cast_emoji(emoji) if emoji else None
```

Reply:
seems smoother in my opinion